server/bundler/i18n-calypso-loader.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Webpack loader used to inject Webpack code into `i18n-calypso` module to
+ * load asyncronously the current locale.
+ */
+
+module.exports = function( content ) {
+	this.cacheable && this.cacheable();
```

Comment:
this line is no longer necessary in webpack2+ since cacheable is [enabled by default](https://webpack.js.org/api/loaders/#this-cacheable)

Reply:
This comment has not been addressed @amanya.

Reply:
I forgot about it, it is removed now